client/devdocs/design/index.jsx

```diff
@@ -85,22 +85,27 @@ let DesignAssets = React.createClass( {
 	},
 
 	render() {
-		const { componentsUsageStats = {} } = this.props;
+		const { componentsUsageStats = {}, component } = this.props;
+		const { filter } = this.state;
+
 		return (
-			<div className="design-assets" role="main">
-				{
-					this.props.component
+			<div className="design" role="main">
```

Comment:
This breaks the main column width. Should switch to use `<Main>`, and you probably need to clean up styles associated with `design-assets`.